docs/quick-start/transmitters/lua-howto.md

```diff
@@ -191,6 +191,10 @@ Available Options:
 
     * Alternate between the two Antennas.
 
+!!! warning "WARNING"
+	When using a Gemini TX with a non-Gemini receiver, you should set the TX in either Switch or Single Antenna (Ant1/Ant2) Mode.
```

Comment:
Please add the reason why it is so.
Maybe add that the other RF chip will produce noise and increase the noise floor while also consuming power, etc.

Reply:
Yes, this is a nice thing to add. 
Please check if the new update fits ;)